scripts/prohibit-undefined-property-keys-in-kt-files

```diff
@@ -0,0 +1,22 @@
+ #!/usr/bin/env bash
+
+set -e -o pipefail -u
+
+self_dir=$(cd "$(dirname "$0")" &>/dev/null; pwd -P)
+source "$self_dir"/utils.sh
+
+bundle=frontendResourceBundles/src/main/resources/GitMacheteBundle.properties
+[[ -f $bundle ]] || die "Bundle file ($bundle) has been (re)moved"
+
+kt_files_with_used_keys=$(git grep -o '\".*\.GitMachete\..*\"' -- '*.kt')
+
+exit_code=0
+for file_with_key in $kt_files_with_used_keys; do
+  file=$(echo $file_with_key | cut -f1 -d:)
```

Comment:
Up to you: this can as well be a nested loop like:
 ```
for file in $(git ls-files '*.kt'); do
  for key in $(git grep -ho ... -- "$file"); do ...
```

Reply:
(With no need for parsing git grep's output)